src/topics/engine/rules/tests/section-regex.test.ts

```diff
@@ -0,0 +1,477 @@
+    {
+      section: 8,
+      input: 'contrôles',
+      expected: false,
+    },
+    {
+      section: 8,
+      input: "del'exposition",
+      expected: false,
+    },
+    {
+      section: 8,
+      input: 'protectionindividuelle',
+      expected: true,
+    },
+    {
+      section: 8,
+      input: 'protection',
+      expected: false,
+    },
+    {
+      section: 8,
+      input: 'individuelle',
+      expected: false,
+    },
+    {
+      section: 9,
+      input: 'propriétésphysiquesetchimiques',
+      expected: true,
+    },
+    {
+      section: 9,
+      input: 'proprietésphysiquesetchimiques',
+      expected: true,
+    },
+
+    {
+      section: 9,
+      input: 'propriétesphysiquesetchimiques',
+      expected: true,
+    },
+    {
+      section: 9,
+      input: 'proprietesphysiquesetchimiques',
+      expected: true,
+    },
+    {
+      section: 9,
+      input: 'propriétés',
+      expected: false,
+    },
+    {
+      section: 9,
+      input: 'proprietés',
+      expected: false,
+    },
+
+    {
+      section: 9,
+      input: 'propriétes',
+      expected: false,
+    },
+    {
+      section: 9,
+      input: 'proprietes',
+      expected: false,
+    },
+    {
+      section: 9,
+      input: 'physiques',
+      expected: false,
+    },
+    {
+      section: 9,
+      input: 'chimiques',
+      expected: false,
+    },
+    {
+      section: 9,
+      input: 'physiques et chimiques',
+      expected: false,
+    },
+    {
+      section: 10,
+      input: 'stabilitéetréactivité',
+      expected: true,
+    },
+    {
+      section: 10,
+      input: 'stabiliteetréactivité',
+      expected: true,
+    },
+    {
+      section: 10,
+      input: 'stabilitéetreactivité',
+      expected: true,
+    },
+    {
+      section: 10,
+      input: 'stabiliteetreactivité',
+      expected: true,
+    },
+    {
+      section: 10,
+      input: 'stabilitéetréactivite',
+      expected: true,
+    },
+    {
+      section: 10,
+      input: 'stabiliteetréactivite',
+      expected: true,
+    },
+    {
+      section: 10,
+      input: 'stabilitéetreactivite',
+      expected: true,
+    },
+    {
+      section: 10,
+      input: 'stabiliteetreactivite',
+      expected: true,
+    },
+    {
+      section: 10,
+      input: 'stabilité',
+      expected: false,
+    },
+    {
+      section: 10,
+      input: 'stabilite',
+      expected: false,
+    },
+    {
+      section: 10,
+      input: 'réactivité',
+      expected: false,
+    },
+    {
+      section: 10,
+      input: 'reactivité',
+      expected: false,
+    },
+    {
+      section: 10,
+      input: 'réactivite',
+      expected: false,
+    },
+    {
+      section: 10,
+      input: 'reactivite',
+      expected: false,
+    },
+  ])('$input for section $section should return $expected', ({ section, input, expected }: { section: number; input: string; expected: boolean }) => {
+    expect(!!input.match(SECTIONS_REGEX[section])).toEqual(expected);
```

Comment:
```suggestion
    expect(input.test(SECTIONS_REGEX[section])).toEqual(expected);
```

comme on la teste :P

Reply:
Ça marche pas avec les accents 🤷‍♂️

Reply:
est ce que t'as essayé l'inverse SECTIONS_REGEX[section].test(input) ?